### PR TITLE
Group list

### PR DESCRIPTION
**Pull request by MilanPospisil**
Test for paging, sorting and filtering of Group list page.

### CODE REVIEW

File: test/cypress/integration/group_list.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    items = sortBy(items, 'name');
+  });
+
+  beforeEach(() => {
+    cy.login(adminUsername, adminPassword);
+  });
+
+  it('table contains all columns', () => {
+    cy.contains('Group');
```
Comment (himdel) [inline test/cypress/integration/group_list.js:28]: we're on the Group list screen.. there will always be *something* that contains "Group"

If the idea is to make sure we have all the columns, what about...

`cy.contains('tr[aria-labelledby="headers"] th', 'Group');`

Comment (himdel) [reply test/cypress/integration/group_list.js:28]: Also, there is no navigation. This is probably finding "Group" in the menus.

Navigate to the right screen first, `cy.menuGo("User Access > Groups")` should do the trick :)

File: test/cypress/integration/group_list.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  });
+
+  it('items are sorted alphabetically and paging is working', () => {
+    cy.visit('/ui/group-list');
+    cy.get('.body:first').contains(items[0].name);
```
Comment (himdel) [inline test/cypress/integration/group_list.js:33]: `.body` is the section containing the whole table. There is always just one.

It looks like the intent was to check the first line in the table contains this, so...

`cy.contains('tbody tr:first td:first', items[0].name);`

File: test/cypress/integration/group_list.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    cy.get('.body:first').get('[aria-label="Go to next page"]:first').click();
+    cy.get('.body:first').contains(items[10].name);
+
+    cy.get('.body:first').get('[aria-label="Go to next page"]:first').click();
+    cy.get('.body:first').contains(items[20].name);
+  });
+
+  it('sorting is working', () => {
+    cy.visit('/ui/group-list');
+    cy.get('.body:first').get('[data-cy="sort_name"]').click();
```
Comment (himdel) [inline test/cypress/integration/group_list.js:44]: presumably..

```
cy.get('[data-cy="sort_name"]').click();
cy.contains('tbody tr:first td:first', items[20].name);
cy.contains('tbody tr td', items[0].name).should('not.exist');
```

```

File: test/cypress/integration/group_list.js (hunk 1 above)
Comment (MilanPospisil) [reply test/cypress/integration/group_list.js:28]: That is totally obvious, I forgot :-) And it finds group substring even in the table :-D

File: test/cypress/integration/group_list.js (hunk 2 above)
Comment (MilanPospisil) [reply test/cypress/integration/group_list.js:33]: Ok, I think I had one issue with it in the past, but that was maybe some exception.

File: test/cypress/integration/group_list.js (hunk 3 above)
Comment (MilanPospisil) [reply test/cypress/integration/group_list.js:44]: You are right, it is good idea to test if the item is right in the first line. But the item, that should not existed, should be tested in whole page, not only in first row, so I leave the contains in the whole body.

**Pull request merged**